README.md

```diff
@@ -276,6 +276,29 @@ func HandleRequest(ctx context.Context, name string) (string, error) {
 }
 ```
 
+## Integrations
```

Comment:
Maybe we can add title as `fasthttp instrumentation`?

Reply:
here: https://github.com/nicolascb/aws-xray-sdk-go/commit/177c39ea7ded5b9dd1a5be8ec48d801f9fa12ea1